packages/input-output-logger/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (opts) => {
+  const defaults = {
+    logger: console.log
+  }
+
+  const options = Object.assign({}, defaults, opts)
+
+  return ({
+    before: (handler, next) => {
+      if (typeof options.logger === 'function') {
+        options.logger(handler.event)
+      }
+
+      return next()
+    },
+    after: (handler, next) => {
+      if (typeof options.logger === 'function') {
+        options.logger(handler.response)
```

Comment:
Same as previous comment, maybe we should make clear that the log line concerns the response.